cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/EnergyConsumerConversion.java

```diff
@@ -26,20 +26,12 @@ public EnergyConsumerConversion(PropertyBag ec, Context context) {
     public void convert() {
         LoadType loadType = id.contains("fict") ? LoadType.FICTITIOUS : LoadType.UNDEFINED;
         LoadAdder adder = voltageLevel().newLoad()
-                .setP0(p0())
-                .setQ0(q0())
+                .setP0(powerFlow().p())
+                .setQ0(powerFlow().q())
                 .setLoadType(loadType);
         identify(adder);
         connect(adder);
         Load load = adder.add();
         convertedTerminals(load.getTerminal());
     }
-
-    private double p0() {
-        return p.containsKey("pfixed") ? p.asDouble("pfixed") : powerFlow().p();
```

Comment:
We have lost the ability to read initial values for `(p0, q0)` from `(pfixed, qfixed)`, that for energy consumers could be a last resort when no powerFlow can be defined from SSH or SV values. Not critical, but we could write `setP0(powerFlow().defined() ? powerFlow().p() : p.asDouble("pfixed"))`.

Reply:
Corrected